head/src/test/kotlin/io/qalipsis/core/head/campaign/states/FactoryAssignmentStateTest.kt

```diff
@@ -272,9 +273,12 @@ internal class FactoryAssignmentStateTest : AbstractStateTest() {
         val newState = state.abort(AbortRunningCampaign())
 
         // then
-        assertThat(newState).isInstanceOf(AbortingState::class).all {
+        assertThat(newState).isInstanceOf(DisabledState::class).all {
```

Comment:
The tests for the new cases that return the abort state with all the factories or just part of them are missing.
- same for all the classes -